.github/workflows/ontobot.yml

```diff
@@ -0,0 +1,82 @@
+name: Create new pull request
+
+on:
+  workflow_dispatch:
+  issues:
+    types: [ opened, edited ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    container: obolibrary/odkfull:v1.4.1
+    strategy:
+      matrix:
+        python-version: ["3.9"]
+        os: [ ubuntu-latest ]
+
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Return issue number
+        id: gh-script-issue
+        uses: actions/github-script@v6
+        with:
+          # github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const issue_number = context.issue.number
+            const repo = context.repo.owner+"/"+context.repo.repo
+            return issue_number
+      
+      - name: Return repository name
+        id: gh-script-repo
+        uses: actions/github-script@v6
+        with:
+          # github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const repo = context.repo.owner+"/"+context.repo.repo
+            return repo
+
+      - name: Set branch name
+        id: vars
+        run: |
+          echo "resource=src/ontology/cl-edit.owl" >> $GITHUB_ENV
+          echo "branch-name=kgcl_automation_"${{ steps.gh-script-issue.outputs.result }} >> $GITHUB_ENV
+      
+      - name: Get jar & create alias command.
+        run: wget https://github.com/gouttegd/kgcl-java/releases/download/kgcl-0.2.0/kgcl-robot-standalone-0.2.0.jar -O kgcl-robot.jar
```

Comment:
This is downloading a full-fledged ROBOT archive (nearly ~90 MB). It’s a shame given that the plugin version is only ~2 MB and ROBOT now has support for plugins.

Reply:
I’ll open a ticket on [ontobot-change-agent](https://github.com/hrshdhgd/ontobot-change-agent) to discuss how the agent could be updated to use a ROBOT plugin when available.

Reply:
The ticket is here: https://github.com/hrshdhgd/ontobot-change-agent/issues/67

Reply:
In the meantime, I have no objection to start using ontobot-change-agent without waiting for that change to be implemented. It can always be updated later.